src/main/webapp/index.html

```diff
@@ -117,38 +117,38 @@ <h3 class="panel__section-heading">Calendar</h3>
           <p class="panel__section-subheading">Selected Calendars:
           <div class="panel__content">
             <div class="panel__content-entry">
-              <p class="panel__text-icon u-text-calendar">M:
+              <p class="panel__text-icon u-text-calendar" id="day-0">
```

Comment:
Instead of hardcoding a "content-entry" for every day of the work week, you should generate these in the Javascript. That way, both the number of work days and the start of the work week can be dictated entirely by the server response.

Reply:
I agree that having one source for the data, and not hardcoding things would be a good way to avoid errors due to un-synchronized number of days. However, having clearly defined classes for these entries allows easy css formatting, that would not be as obvious if the entire panel was set from the JS script.

Reply:
You are using the same classes for each day, so I don't think there would be a problem with generating this from Javascript.